### PR TITLE
[docs] updated `graphql` data source docs

### LINKED ISSUE #3094: [docs]: Explain how to include variables in GraphQL queries
### Summary

The doc page for GraphQL doesn't explain how to include variables in a GraphQL query. I asked on the Slack channel and somebody kindly explained that you can inject parameters directly into the query using Javascript, like this:
```
query Query {
  characters(page: {{components.textinput1.value}}, filter: {name: "Morty"}) {
    info {
      count
    }
    results {
      name
    }
  }
  location(id: 1) {
    id
  }
  episodesByIds(ids: [1, 2]) {
    id
  }
}
```

This makes perfect sense and I think it would be great to include it in the documentation.

### Issue Type

Documentation feature/update

### The entire URL of the documentation with the issue

https://docs.tooljet.com/docs/data-sources/graphql/

### Steps to reproduce the issue

_No response_

### Additional Information

It makes it easier to understand how to include variables in a GraphQL API query.

### If the issue is confirmed, would you be willing to submit a pull request?

No

### Code of Conduct

- [X] I agree to follow the ToolJet Code of Conduct

### PR DESCRIPTION
**Pull request by withshubh**
### changes
- ~how to include variables in graphql queries~
- images update

fixes #3094

### CODE REVIEW

File: docs/docs/data-sources/graphql.md [hunk 1]
```diff
@@ -23,19 +22,52 @@ The following optional parameters are also supported:
...
 
 :::tip
 Query results can be transformed using transformations. Read our transformations documentation to see how: [link](/docs/tutorial/transformations)
-:::
\ No newline at end of file
+:::
+
+## Including variables in GraphQL queries
+
+You can inject parameters directly into the query using Javascript.
```
Comment (gsmithun4) [inline docs/docs/data-sources/graphql.md:51]: we have added `variables` field, this is no more required.

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: docs/docs/data-sources/graphql.md (hunk 1 above)
Comment (withshubh) [reply docs/docs/data-sources/graphql.md:51]: Awesome! 
@Navaneeth-pk I think we can now close this PR and the issue

Comment (withshubh) [reply docs/docs/data-sources/graphql.md]: Thanks! removed these changes.

**Review by gsmithun4 — APPROVED** [review]
LGTM

**Pull request merged**